LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2017 The Kubernetes Authors
```

Comment:
We shouldn't change this, I thought... e.g. https://github.com/kubernetes/kubernetes/commit/d30db1f9a915aa95402e1190461469a1889d92be